### PR TITLE
Removed duplicated code in docker-compose.yml

### PR DESCRIPTION
**Pull request by Nish251103**

### CODE REVIEW

**Comment by riibeirogabriel** [comment]
Sorry, it doesn't seems good, you remove so much lines, but you don't create a module with this lines to replace the removed lines, and you need update the docker-compose.yml version to make this feature.

**Comment by riibeirogabriel** [comment]
@Nish251103 Do you yet have interest in this issue?

**Comment by Nish251103** [comment]
I am trying to do it. I have no knowledge about docker-compose

**Comment by Nish251103** [comment]
I tried it again 
Kindly have a look
Thank You

File: docker-compose.yml [hunk 1]
```diff
@@ -1,3 +1,18 @@
...
+def deploys():
+    deploy:
+        replicas: 1
+        restart_policy:
+            condition: on-failure
+        placement:
+            constraints:
+                - "node.role==worker"
+
+def networkd():
```
Comment (learningOrchestraAccount) [inline docker-compose.yml:11]: A name more descrictive to this network like spark_network sounds better?

**Review by learningOrchestraAccount — COMMENTED** [review]
@Nish251103 thanks for spend your time with this project, i have commented in your PR, take a look and give me a feedback.

**Comment by riibeirogabriel** [comment]
@Nish251103 i have used the learningOrchestra account to make this review, please if you want notify, notify in my personal account.

**Comment by Nish251103** [comment]
@riibeirogabriel 
1.query -> I have no knowledge of it sorry can't help with this query

2.query - > it's a inbuilt function so I don't think so an update is needed 
I am a student and was taught the module feature with this function syntax at school.

3 query - > okay I can do that on

Thank you 
Should I do the needful what I feel is I don't have much knowledge about it but am eager to help
You will have to assign it to someone who really knows about it

**Comment by riibeirogabriel** [comment]
@Nish251103 no problems, and thanks to waste your time with this project, i can get up some issue in python to you, of course case you want.

**Comment by Nish251103** [comment]
@riibeirogabriel 
It is never a waste of time it helped me know what i am lacking so i will study more about this and would be ready to deal with it.
yeah if you have a python related task i would be happy to work on but as i am new to open-source dont have hopes that i would do it perfectly sorry for that.
I ensure you i will try my best

**Pull request closed without merging**